cmd/vcsim/main.go

```diff
@@ -64,5 +69,21 @@ func main() {
 		log.Fatal(err)
 	}
 
-	model.Service.NewServer()
+	if *isTLS {
+		model.Service.TLS = new(tls.Config)
+		if *cert != "" {
+			c, err := tls.LoadX509KeyPair(*cert, *key)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			model.Service.TLS.Certificates = []tls.Certificate{c}
+		}
+	}
+
+	s := model.Service.NewServer()
+
+	fmt.Printf("GOVC_URL=%s", s.URL)
```

Comment:
Yes, stdout will only contain env variables, for use by out-of-process clients.  For example, you can run with `-httptest.serve 127.0.0.1:0` and the value of GOVC_URL contains the randomly chosen port.